posix-mapper/src/main/webapp/capabilities.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<vosi:capabilities
+    xmlns:vosi="http://www.ivoa.net/xml/VOSICapabilities/v1.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:vs="http://www.ivoa.net/xml/VODataService/v1.1">
+
+  <capability standardID="http://www.opencadc.org/std/posix#user-mapping-1.0">

```

Comment:
the constants in Standards use version 0.1 for prototype so that should be used here (and below)